### PR TITLE
[57008] Open Project Darkmode: pressing buttons / flash colour gets too bright

### PR DESCRIPTION
**Pull request by bsatarnejad**
# What are you trying to accomplish?
I want to change background color of toolbar buttons in active state for both dark and light modes.

## Screenshots

Before
Screenshot 2024-08-09 at 11 20 02

After
Screenshot 2024-08-09 at 11 21 28

# What approach did you choose and why?
I replaced some custom css variables with primer variables for toolbar items. We want to remove our custom css variables and use primer variables instead as much as we can.

# Ticket
https://community.openproject.org/work_packages/57008/activity

# Merge checklist

- [ ] Added/updated tests
- [ ] Added/updated documentation in Lookbook (patterns, previews, etc)
- [X] Tested major browsers (Chrome, Firefox, Edge, ...)

### CODE REVIEW

File: frontend/src/global_styles/layout/_toolbar.sass [hunk 1]
```diff
@@ -165,15 +165,15 @@ $nm-color-success-background: #d8fdd1
...
 
   .button,
   .toolbar-input--affix
-    background: var(--toolbar-item--bg-color)
+    background: var(--button-default-bgColor-rest)
 
     &.-pressed,
     &:active
-      background: var(--toolbar-item--bg-color-pressed)
+      background: var(--control-transparent-bgColor-active)
```
Comment (HDinger) [inline frontend/src/global_styles/layout/_toolbar.sass]: I'd suggest to use `--button-default-bgColor-active` here instead

**Review by HDinger — COMMENTED** [review]
Just one minor remark. Other than that it is good to merge

File: frontend/src/global_styles/layout/_toolbar.sass (hunk 1 above)
Comment (bsatarnejad) [reply frontend/src/global_styles/layout/_toolbar.sass]: Hi @HDinger 
I got the variable from GitHub toolbar items styles. Agree this variable is better 👍

Comment (HDinger) [reply frontend/src/global_styles/layout/_toolbar.sass]: I see the `--button-default-bgColor-active` in GitHub as well

Bildschirmfoto 2024-08-09 um 13 21 46

**Review by HDinger — APPROVED** [review]

**Pull request merged**